yt_dlp/extractor/netverse.py

```diff
@@ -16,23 +14,23 @@ class NetverseBaseIE(InfoExtractor):
         'watch': 'watchvideo',
         'video': 'watchvideo',
         'webseries': 'webseries',
+        'season': 'webseason_videos',
     }
 
-    def _call_api(self, url, query={}):
-        display_id, sites_type = self._match_valid_url(url).group('display_id', 'type')
+    def _call_api(self, input_data, query={}, season='',
+                  custom_id=None, force_endpoint_type='auto', input_type='url'):
```

Comment:
This doesn't really make sense. Try:
```suggestion
    def _call_api(self, url=None, slug=None, endpoint=None, query={}, season=''):
        assert (url is None and slug) or (slug is None and url)
        ...
```
I don't see the point of `custom_id`

Reply:
I use `custom_id` only to force `video_id` in `self._download_json` to keep consistent `id`. The actual `id` is getting used is `slug` from `slug_sample`